constants/profile/user.js

```diff
@@ -0,0 +1,7 @@
+export default {
```

Comment:
This should be in the store as it can change through user interactions and needs to be shared among multiple components

Reply:
True. Changing